### PR TITLE
specify region to prevent sbt error

### PR DESCRIPTION
**Pull request by timcharper**
JIRA Issues: MARATHON-7917

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -108,6 +108,7 @@ lazy val commonSettings = testSettings ++
     s3("downloads.mesosphere.io/maven")
   )),
   s3credentials := new EnvironmentVariableCredentialsProvider() | InstanceProfileCredentialsProvider.getInstance(),
+  s3region := com.amazonaws.services.s3.model.Region.US_West_2,
```
Comment (meln1k) [inline build.sbt]: Is it the right region?

Comment (meln1k) [reply build.sbt]: It used to be a different one:

https://github.com/ohnosequences/sbt-s3-resolver/blob/v0.8.0/src/main/scala/SBTS3Resolver.scala#L51

**Comment by kensipe** [comment]
#5811 seems to be the same change... oddly these 2 PRs reference different JIRAs.

**Comment by timcharper** [comment]
Dude, I give up.

**Pull request closed without merging**